app/views/landing.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+<title>Subscribe to the Scenic today!</title>
+<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Playfair+Display" type="text/css">
+<link href="https://fonts.googleapis.com/css?family=Muli" rel="stylesheet" type="text/css">
+<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700" type="text/css">
+<link rel="stylesheet" href="/styles.css" type="text/css">
+<link rel="icon" type="image/png" href="/icons/icon-2x.png">
+<meta name="theme-color" content="#37dc8c">
+<link rel="apple-touch-icon" href="/icons/icon-2x.png">
+<meta name="msapplication-TileImage" content="/icons/icon-2x.png">
+<meta name="msapplication-TileColor" content="#37dc8c">
+<style>
+  .button-container {
+    font-size: 16px;
+    padding: 10px;
+    text-align: center;
+    background-color: #4CAF50;
+    color: white;
+    border: solid 1px #a6d8a8;
+    margin: auto;
+  }
+</style>
+</head>
+<body>
+  <div class="main-body">
+    <header>
+      <a class="brand-header">
+        <div class="brand-logo"></div>
+        <div class="brand-name">The Scenic</div>
+      </a>
+    </header>
+    <main role="main">
+      <div class="container">
+        <h1 class="heading">Scenic Subscription offer</h1>
+        <div class="hr"></div>
+        <p class="text">
+          Enjoy 50 years of free Scenic content.  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas nunc felis, iaculis eget arcu sed, venenatis tincidunt dui. Mauris semper neque vitae quam auctor convallis. Suspendisse condimentum sapien eget porta.</p>
+        </p>        
+        <button class="button-container" id="subscribeButton" disabled>Click to claim offer</button>
+    </div>
+  </div>
+<script async subscriptions-control="manual" src="<% swgJsUrl %>"></script>
+<script>
+function createPublisherAccount() {
+    return new Promise(function(resolve) {
+      setTimeout(() => {
+        resolve('true');
+      }, 2000);
+    });
+}
+function start() {
+  (self.SWG = self.SWG || []).push(function(subscriptions) {
+    subscriptions.init('<% publicationId %>');
+    subscriptions.setOnEntitlementsResponse(function(entitlementsPromise) {
+      console.log('Fetching entitlements');
+      entitlementsPromise.then(entitlements => {
+        console.log('got entitlements: ', entitlements);
+        if (entitlements && entitlements.enablesAny()) {
+            //Redirect to home page or account page
```

Comment:
Let's do a consistent indent. Here it uses 2sp and 4sp.

Reply:
done